graas/static/index.xml

```diff
@@ -18,14 +18,25 @@
         <li role="presentation"><a href="#">About</a></li>
       </ul>
     </nav>
-    <h3 class="text-muted">GRaaS</h3>
+    <h3 class="text-muted" xmlns:t="http://twistedmatrix.com/ns/twisted.web.template/0.1">
+      <a href="/"><span t:render="name"></span></a>
+    </h3>
+  </div>
+
+  <div xmlns:t="http://twistedmatrix.com/ns/twisted.web.template/0.1">
+    <div t:render="message"></div>
   </div>
 
   <div class="jumbotron">
-    <form action="/action/press" method="post">
+    <form action="/" method="post">
```

Comment:
Instead of changing this to POST to the homepage, I think we should keep this as a POST to `/action/press`, and have that endpoint return a redirect.

That way, if someone refreshes the page after the POST request, it doesn't trigger the action again.